### ISSUE TITLE
Support without Access Keys if run on EC2 Instance

### ISSUE THREAD

**Issue by mkanchwala** [gmdutra/aws-report#3]
Summary of the Issue is If I want to use it without Access Keys on EC2 instance itself to get reports on other resources.

**Comment by gmdutra** [comment]
Hi,
You attached a iam role in the instance?

**Comment by mkanchwala** [comment]
Yes If we attach an IAM role and do the report generation will it work? I don't want to generate the Access keys and secrets neither my organization audit policy allows it.
